hummingbot/connector/exchange/bittrex/bittrex_constants.py

```diff
@@ -0,0 +1,60 @@
+# A SINGLE POINT OF TRUTH FOR BITTREX
+from hummingbot.core.api_throttler.data_types import RateLimit
+from hummingbot.core.data_type.in_flight_order import OrderState
+
+EXCHANGE_NAME = "Bittrex"
+PING_TIMEOUT = 10.0
+MAX_RETRIES = 20
+MESSAGE_TIMEOUT = 30.0
+SNAPSHOT_TIMEOUT = 10.0
+
+BITTREX_REST_URL = "https://api.bittrex.com/v3"
+BITTREX_WS_URL = "https://socket-v3.bittrex.com/signalr"
+DEFAULT_DOMAIN = ""
```

Comment:
If there is no need for this, lets remove it

Reply:
Removed.